trajectory/src/main/java/io/github/agentwise/swarmview/trajectory/applications/trajectory/StraightLineTrajectory4D.java

```diff
@@ -72,10 +72,10 @@ private StraightLineTrajectory4D(
         checkArgument(
                 velocityCutoffTimePercentage <= 1 && velocityCutoffTimePercentage > 0,
                 "Velocity cutoff percentage should represent a percantage between 0 and 1.");
-        checkArgument(
-                velocity <= MAX_ABSOLUTE_VELOCITY,
-                "The provided enterVelocity should be smaller than BasicTrajectory"
-                        + ".MAX_ABSOLUTE_VELOCITY");
+//        checkArgument(
```

Comment:
It's for the wiggle effect. We need to generate very high velocity (15m/s) for the wiggle.